systemFlake.nix

```diff
@@ -106,9 +119,5 @@ otherArguments
   // optional defaultAppBuilder { defaultApp = defaultAppBuilder pkgs; }
   // optional devShellBuilder { devShell = devShellBuilder pkgs; }
   // optional checksBuilder { checks = checksBuilder pkgs; }
-)
-
-  // {
-  nixosConfigurations = nixosConfigurations // (builtins.mapAttrs nixosConfigurationBuilder nixosProfiles);
-}
+) // (mapAttrs' configurationBuilder configProfiles)
```

Comment:
Not necessarily against the `nixosConfigurations //` part, but I'm not sure the best way to integrate with this.

Reply:
considering https://github.com/gytis-ivaskevicius/flake-utils-plus/pull/15#issuecomment-814995499, `nixosConfiguations` is not going to work anyway. I would say just drop the escape hatch entirely, flake-utils-plus is general enough, where its pretty unecessary.